.coveragerc

```diff
@@ -0,0 +1,5 @@
+# .coveragerc to control coverage.py
```

Comment:
is this supposed to affect the coverage locally or the coverage in codecov. If it is locally, u should try to create 2 reports with and without this file and see if there is difference

Reply:
Should be " coverage in codecov".

Reply:
But this may not be the correct way to configure

Reply:
related issue: https://github.com/codecov/codecov-python/issues/107